src/Features/Core/Portable/GenerateMember/GenerateVariable/AbstractGenerateVariableService.GenerateParameterCodeAction.cs

```diff
@@ -0,0 +1,52 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.CodeAnalysis.AddParameter;
+using Microsoft.CodeAnalysis.CodeActions;
+
+namespace Microsoft.CodeAnalysis.GenerateMember.GenerateVariable
+{
+    internal partial class AbstractGenerateVariableService<TService, TSimpleNameSyntax, TExpressionSyntax>
+    {
+        private class GenerateParameterCodeAction : CodeAction
+        {
+            private readonly Document _document;
+            private readonly State _state;
+            private readonly bool _includeOverridesAndImplementations;
+
+            public GenerateParameterCodeAction(Document document, State state, bool includeOverridesAndImplementations)
+            {
+                _document = document;
+                _state = state;
+                _includeOverridesAndImplementations = includeOverridesAndImplementations;
+            }
+
+            public override string Title
+            {
+                get
+                {
+                    var text = _includeOverridesAndImplementations ? FeaturesResources.Generate_parameter_0_and_overrides_implementations : FeaturesResources.Generate_parameter_0;
```

Comment:
line too long. wrap like:

```c#
var text = _includeOverridesAndImplementations
    ? FeaturesResources.Generate_parameter_0_and_overrides_implementations
    : FeaturesResources.Generate_parameter_0;
```

A good rule of thumb is: too wide for github? wrap.